cmake/FindTcl.cmake

```diff
@@ -0,0 +1,9 @@
+include(${CMAKE_MODULE_PATH}/pkg_helper.cmake)
+find_pkg_for_comdb2(Tcl
```

Comment:
I didn't have tcl.h on my system - it passed cmake and failed at compile time - this step should fail if not found (or tcl bindings can be optional and only build if tcl exists).

Reply:
I'll figure out how to make the necessary changes for CMake; however, I may need your help in testing them as most (other) systems will probably have the Tcl stuff installed?

Reply:
This issue should be fixed now.  Tcl support is not included unless enabled.